packages/dashboard/src/components/Card/animations/DonutChart.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+
+import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+
+import { getStyles, IDonutChartStyles } from './DonutChart.style';
+import { classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
+
+export interface IDonutChartProps {}
+
+export class DonutChart extends React.Component<IDonutChartProps> {
+  constructor(props: IDonutChartProps) {
+    super(props);
+  }
+
+  public render(): JSX.Element {
+    const getClassNames = classNamesFunction<IDonutChartProps, IDonutChartStyles>();
+    const classNames = getClassNames(getStyles, {});
+
+    const donutChartLoadingBaseClassName = mergeStyles(getStyles().donutChartLoadingBase, getStyles().donutChartLoadingSegment);
```

Comment:
Would be great to avoid repeatedly calling `getStyles`. Consider either using `classNames.donutChartLoadingBase`, `classNames.donutChartLoadingSegment`, etc. or, in DonutChart.styles, combining the styles for these class names into one class name.

Reply:
The class donutChartLoadingBase is base class it will be used in another classes while render that's why we use mergeStyles. each class has it's own purpose.

Reply:
Could you clarify? It doesn't seem like mergeStyles needs to be called here.